test/e2e_node/dra_test.go

```diff
@@ -656,6 +695,10 @@ func createTestObjects(ctx context.Context, clientSet kubernetes.Interface, node
 			RestartPolicy: v1.RestartPolicyNever,
 		},
 	}
+	if podName == "pausepod" {
```

Comment:
@bart0sh: I considered adding another parameter to `createTestObjects` but decided against it because it would have affected all other call sites. Up for debate...

Reply:
It looks ok to me, considering the alternative.

Reply:
So let's keep it. I added tests for it.